### PR TITLE
fix: Fix the Bg color of the sign In button after clicking Fix No #12773

### PR DESCRIPTION
**Pull request by suprabhu77**
## What does this PR do?

Fixes # (issue)
Now User will be able to see the login/Sing In Text even upon the click event. after the click the button bg will be lightly graded.

https://github.com/calcom/cal.com/assets/65330670/8cd37e78-1598-4e0d-923f-efb100d602ce

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Tests (Unit/Integration/E2E or any other test)
- [ ] This change requires a documentation update

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by suprabhu77** [comment]
@PeerRich Can you help me in reviewing the code, This is my first Open source PR. Looking forward to contribute more to cal.com.
Happy Open Sourcing.

Regards
Sumantha

**Review by sean-brydon — CHANGES_REQUESTED** [review]
Hi - thanks for your contribution. 

Try to use tailwind to implement this fix.

The issue here is there is a typo when setting the CSS variable [here](https://github.com/calcom/cal.com/pull/12833/files#diff-d375d403d178baa1229f926b7d2de762c1e2e7e4e2c013834270e1f961ff63c1R176)

`[--cal-brand-subtle:9CA3AF]` -> `[--cal-brand-subtle:#9CA3AF]`

**Comment by suprabhu77** [comment]
@sean-brydon Will look into this and update the PR Soon.
Thank you for reviewing.

**Comment by suprabhu77** [comment]
@sean-brydon  I have updated the PR, You were right, it was regarding the typo error.
Thank you for the help.
Please help me in contributing to cal.com

**Review by sean-brydon — DISMISSED** [review]

**Comment by suprabhu77** [comment]
@sean-brydon 
Why is linters workflow is getting failed over there?
How can i again hit the workflow?

**Comment by suprabhu77** [comment]
@sean-brydon 
Please help me in this PR, Lint error is getting failed if I make changes to #9CA3AF
Something is wrong in that.

**Comment by suprabhu77** [comment]
@sean-brydon 
I have made the changes and now its working.
Can you please approve the request?

File: apps/web/pages/auth/login.tsx [hunk 1]
```diff
@@ -229,7 +229,7 @@ inferSSRProps<typeof _getServerSideProps> & WithNonceProps<{}>) {
                 type="submit"
                 color="primary"
                 disabled={formState.isSubmitting}
-                className="w-full justify-center">
+                className="w-full justify-center bg-white/70">
```
Comment (Udit-takkar) [inline apps/web/pages/auth/login.tsx:232]: https://github.com/calcom/cal.com/assets/53316345/de6d7d54-72c7-41f9-9245-9ee6815be1b5

This doesn't fix the issue

**Review by Udit-takkar — DISMISSED** [review]
LGTM. I have pushed the fix

**Comment by suprabhu77** [comment]
@Udit-takkar I tried changing the tailwind css in the class name, But ESlint workflow was throwing some error.
So I tried changing in the Button Classname

**Comment by Udit-takkar** [comment]
@suprabhu77 install prettier and configure it to format on save in vscode

**Comment by suprabhu77** [comment]
@Udit-takkar 
Can you help me in this Lint Error? I have prettier and there is no error in local setup.

File: .vscode/settings.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "typescript.tsdk": "node_modules/typescript/lib",
   "editor.formatOnSave": false,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```
Comment (PeerRich) [inline .vscode/settings.json:5]:
```suggestion
    "source.fixAll.eslint": true
```

**Review by PeerRich — DISMISSED** [review]

**Comment by Udit-takkar** [comment]
> @Udit-takkar Can you help me in this Lint Error? I have prettier and there is no error in local setup.

checking

**Review by Udit-takkar — APPROVED** [review]

**Pull request merged**